vault/token_store.go

```diff
@@ -382,16 +411,30 @@ func (ts *TokenStore) paths() []*framework.Path {
 		{
 			Pattern: "lookup",
 
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixToken,
+			},
+
 			Fields: map[string]*framework.FieldSchema{
 				"token": {
 					Type:        framework.TypeString,
 					Description: "Token to lookup (POST request body)",
 				},
 			},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   ts.handleLookup,
-				logical.UpdateOperation: ts.handleLookup,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: ts.handleLookup,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb: "lookup",
+					},
+				},
+				logical.UpdateOperation: &framework.PathOperation{
+					Callback: ts.handleLookup,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb: "lookup2",
```

Comment:
what does `lookup2` mean? Shouldn't it be differentiated by the type of operation i.e. `Update` vs `Read`

Reply:
Both Update and Read result in exactly the same operation (`handleLookup`). However, we can't have duplicate operation id's, so these need to be deduped. In such cases, I've been using a "2" suffix for lack of a better name. In the libraries we can filter these out with handlebar templates.

```
token-lookup
token-lookup2
```